[2025-03-19T19:07:13.454+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-03-19T19:07:13.523+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v2.get_age scheduled__2024-11-29T00:00:00+00:00 [queued]>
[2025-03-19T19:07:13.551+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_taskflow_Api_v2.get_age scheduled__2024-11-29T00:00:00+00:00 [queued]>
[2025-03-19T19:07:13.555+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 6
[2025-03-19T19:07:13.885+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): get_age> on 2024-11-29 00:00:00+00:00
[2025-03-19T19:07:14.015+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=10024) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-03-19T19:07:14.028+0000] {standard_task_runner.py:72} INFO - Started process 10050 to run task
[2025-03-19T19:07:14.018+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_with_taskflow_Api_v2', 'get_age', 'scheduled__2024-11-29T00:00:00+00:00', '--job-id', '2912', '--raw', '--subdir', 'DAGS_FOLDER/dag_with_taskflow_api.py', '--cfg-path', '/tmp/tmp8k0568f5']
[2025-03-19T19:07:14.038+0000] {standard_task_runner.py:105} INFO - Job 2912: Subtask get_age
[2025-03-19T19:07:15.438+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_with_taskflow_Api_v2.get_age scheduled__2024-11-29T00:00:00+00:00 [running]> on host 63fd4b143e0a
[2025-03-19T19:07:19.447+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_with_taskflow_Api_v2' AIRFLOW_CTX_TASK_ID='get_age' AIRFLOW_CTX_EXECUTION_DATE='2024-11-29T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-29T00:00:00+00:00'
[2025-03-19T19:07:19.510+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-03-19T19:07:19.589+0000] {python.py:240} INFO - Done. Returned value was: 19
[2025-03-19T19:07:20.124+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-03-19T19:07:20.129+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=dag_with_taskflow_Api_v2, task_id=get_age, run_id=scheduled__2024-11-29T00:00:00+00:00, execution_date=20241129T000000, start_date=20250319T190713, end_date=20250319T190720
[2025-03-19T19:07:21.286+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-03-19T19:07:24.188+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2025-03-19T19:07:25.151+0000] {taskinstance.py:3901} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-03-19T19:07:25.186+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
